game/ffmpeg.hh

```diff
@@ -45,14 +44,14 @@ class VideoFifo {
 	}
 	/// Add frame to queue
 	void push(Bitmap* f) {
-		boost::mutex::scoped_lock l(m_mutex);
-		while (m_queue.size() > m_max) m_cond.wait(l);
+		std::unique_lock<std::mutex> l(m_mutex);
+		m_cond.wait(l, [this]{ return m_queue.size() < m_max; });
```

Comment:
Because stdlib condition_variables work like this. The call will only return when the test returns true. The Boost-style API is not available in stdlib.